charts/pulsar/templates/broker-statefulset.yaml

```diff
@@ -166,6 +166,8 @@ spec:
         command: ["sh", "-c"]
         args:
         - >
+          echo $'\n' >> conf/pulsar_env.sh;
+          echo "PULSAR_EXTRA_CLASSPATH=/sn-platform/extraLibs" >> conf/pulsar_env.sh;
```

Comment:
why not add `PULSAR_EXTRA_CLASSPATH` in the config map? Currently, all the fields in configmap are already imported as environment variables.

Reply:
done @sijie PTAL

```
root@sn-platform-pulsar-broker-0:/sn-platform# echo $PULSAR_EXTRA_CLASSPATH
/sn-platform/extraLibs
```